src/librustdoc/html/render/mod.rs

```diff
@@ -1634,6 +1634,18 @@ fn render_impl(
         }
 
         if let Some(ref dox) = cx.shared.maybe_collapsed_doc_value(&i.impl_item) {
+            if render_mode == RenderMode::Normal
+                && trait_.is_none()
+                && i.inner_impl().items.is_empty()
+            {
+                w.write_str(
+                    "<div class=\"item-info\">\
+                    <div class=\"stab unstable\">\
```

Comment:
Why does this use the unstable class?

Reply:
Completely forgot to create a new "empty-impl" class, thanks!